src/renderer/components/main/mainbar/statusbar_word_count.tsx

```diff
@@ -0,0 +1,21 @@
+
+/* IMPORT */
+
+import * as React from 'react';
+import {connect} from 'overstated';
+import Main from '@renderer/containers/main';
+
+/* STATUSBAR WORD COUNT */
+
+const WordCount = ({ content }) => {
```

Comment:
Isn't there a third-party plugin that provides this to codemirror?

Reply:
I wasn't sure how many codemirror plugins you wanted to be reliant on in the core codebase.